cmd/restic/cmd_dump.go

```diff
@@ -50,8 +50,9 @@ Exit status is 12 if the password is incorrect.
 // DumpOptions collects all options for the dump command.
 type DumpOptions struct {
 	restic.SnapshotFilter
-	Archive string
-	Target  string
+	Archive            string
+	Target             string
+	CompressZipArchive bool
```

Comment:
Just `Compress` should be fine. Considering, we can compress a `.tar` after archiving.